packages/lib/models/Note.ts

```diff
@@ -697,6 +691,11 @@ export default class Note extends BaseItem {
 			beforeNoteJson = JSON.stringify(oldNote);
 		}
 
+		if ('is_todo' in o && o.is_todo === 0) {
+			o.todo_due = 0;
+			o.todo_completed = 0;
+		}
```

Comment:
I'm worried about the unintended effects of this. I don't see any reason why todo_completed or todo_due shouldn't be left as they are when is_todo is reset. What if for some weird reason some code somewhere is relying on these properties still being set?

It just feels like leaving them as they are won't cause any problems (calling code can check "is_todo" to know if they should care about todo_due and todo_completed or not). While removing them might.

Reply:
Okay, I've removed this and its related logic